src/go/k8s/tests/e2e/pandaproxy-produce-consume-sasl/00-assert.yaml

```diff
@@ -36,3 +36,7 @@ collectors:
 - type: pod
   selector: app.kubernetes.io/name=redpanda
   tail: -1
+- type: pod
+  namespace: redpanda-system
+  selector: control-plane=controller-manager
+  tail: -1
```

Comment:
Please don't use that. I already removed all occurrences in https://github.com/redpanda-data/redpanda/pull/10122

Reply:
Don't know why I missed this file. Removed.